src/translations/vi.json

```diff
@@ -0,0 +1,21 @@
+{
+  "error-occured": "Đã có lỗi xảy ra khiến cho phần mở rộng không hoạt động như mong muốn: ",
+  "version-update-message": "Bạn đang sử dụng ezpp! v{0}",
+  "changelog": "lịch sử cập nhật",
+  "accuracy": "Độ chính xác",
+  "combo": "Combo",
+  "empty-fc": "(rỗng = Full Combo)",
```

Comment:
this is an outdated translation line and can be removed. speaking of, I should add some tests for translations 🤔